homeassistant/components/zha/strings.json

```diff
@@ -73,26 +104,32 @@
       "invalid_backup_json": "Invalid backup JSON"
     },
     "abort": {
-      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
+      "single_instance_allowed": "Only a single ZHA instance is allowed",
```

Comment:
nit: Since this string was deleted and later re-added at some point, is the change here intended?

Like, do we want to explicitly mention ZHA in this message? The previously used common string is: "Already configured. Only a single configuration possible."